src/settings/panel.tsx

```diff
@@ -0,0 +1,245 @@
+import { IRenderMimeRegistry } from '@jupyterlab/rendermime';
+import { ISettingRegistry } from '@jupyterlab/settingregistry';
+import { FormComponent, IFormRenderer } from '@jupyterlab/ui-components';
+import { JSONExt } from '@lumino/coreutils';
+import { IChangeEvent } from '@rjsf/core';
+import type { FieldProps } from '@rjsf/utils';
+import validator from '@rjsf/validator-ajv8';
+import { JSONSchema7 } from 'json-schema';
+import React from 'react';
+
+import { IDict, instructions } from './instructions';
+import baseSettings from './schemas/base.json';
+import ProviderSettings from './schemas';
+
+const MD_MIME_TYPE = 'text/markdown';
+const STORAGE_NAME = '@jupyterlite/ai:settings';
+const INSTRUCTION_CLASS = 'jp-lite-ai-settings-instruction';
```

Comment:
If we want to follow a similar naming style as in JupyterLab, maybe the following class name could also work?

```suggestion
const INSTRUCTION_CLASS = 'jp-AISettingsInstructions';
```